Utilities/SimpleITKJSONDream3DFilterCreation.py

```diff
@@ -7,14 +7,19 @@
 JSON files are taken from [1].
 Documentation about the JSON files is available here [2].
 Test implementation in SimpleITK: [3]
+ITK can be downloaded here: [4]
 
 Usage:
 
-./SimpleITKJSONDream3DFilterCreation.py  -j ~/Devel/SimpleITK/Code/BasicFilters/json/ -d .. -n
+./SimpleITKJSONDream3DFilterCreation.py  -j ~/Devel/SimpleITK/Code/BasicFilters/json/ -d .. -n -o -I ~/Devel/ITK
```

Comment:
This invocation will fail as it will try to find some templates that do not currently exist. To make it work, one needs to have a "JSON folder" that only contains JSON files that can be converted (this is the short answer).